tests/docker/config/construct-query.sparql

```diff
@@ -74,62 +139,53 @@ CONSTRUCT {
-  ?id  :circuitType        ?circuitType           ;
-       :circuitBase        ?circuitBaseUrlStr        ;
-       :circuitConfigPath  ?circuitConfigPathUrlStr  .
 
-  ?id  :startedAt            ?startedAt               ;
-       :endedAt              ?endedAt                 ;
-       :status               ?status                  ;
-       :circuitName          ?circuitName   ;
-       :circuitBrainRegion   ?circuitBrainRegionBN    .
-                             ?circuitBrainRegionBN    :identifier  ?circuitBrainRegionId    ;
-                                                      :label       ?circuitBrainRegionLabel .
-  ?id  :derivation           ?derivation              .
-                             ?derivation              :identifier      ?entity     ;
-                                                      rdf:type         ?entityType ;
-                                                      :label           ?entityName .
-  ?id  :distribution         ?distribution            .
-                             ?distribution            :label           ?distributionName           ;
-                                                      :encodingFormat  ?distributionEncodingFormat ;
-                                                      :contentUrl      ?distributionContentUrl     ;
-                                                      :contentSize     ?distributionContentSize    .
-  ?id  :image                ?image                   .
-                             ?image                   :identifier      ?image                  ;
-                                                      :about           ?imageAbout             ;
-                                                      :repetition      ?imageRepetition        ;
-                                                      :stimulusType    ?imageStimulusTypeLabel .
-  ?id  :series               ?series                  .
-                             ?series                  :value           ?seriesValue      ;
-                                                      :unit            ?seriesUnit       ;
-                                                      :statistic       ?seriesStatistic  .
+  ## Neuron density
+  ?id  :neuronDensity      ?neuronDensityBN       .
+                           ?neuronDensityBN       :value       ?neuronDensityValue   ;
+                                                  :unit        ?neuronDensityUnit    ;
+                                                  :nValue      ?neuronDensityNValue  ;
+                                                  :label       ?neuronDensityLabel   .
+
+  ## Simulation campaigns
+  ?id  :config               ?campaignConfigBN        .
+                             ?campaignConfigBN        :identifier      ?campaignConfigId   ;
+                                                      :label           ?campaignConfigName .
```

Comment:
Is it possible to get rid of the blank node here? Just using ?campaignConfigId instead. Eg. what is done for species and others